src/scripts/main/toast/index.js

```diff
@@ -1,4 +1,5 @@
 import { extendObj } from "../utils";
+import { callbackify } from "util";
```

Comment:
I'm not sure about this import, but I think it can be removed and then we can merge this PR?

Reply:
Agree. Not sure how it ended up there, will remove ASAP